core/cap-0045.md

```diff
@@ -391,8 +391,41 @@ price <img style="transform: translateY(0.1em); background: white;" src="https:/
 Specifically, <img style="transform: translateY(0.1em); background: white;" src="https://render.githubusercontent.com/render/math?math=%5Cfrac%7Bp_%7Bfull%7D%7D%7Bp_%7Blim%7D%7D%20%3D%20%5Cfrac%7B1%7D%7B(1-2%5E%7B-%5Csigma%7D)%7D">.
 
 It doesn't matter whether you linearly interpolate the asset sold or the asset bought. The two quantities are linearly related according to the budget
-constraint `pX + Y = pX' + Y'`, so linear interpolation of one implies linear
-interpolation of the other.
+constraint <img style="transform: translateY(0.1em); background: white;" src="https://render.githubusercontent.com/render/math?math=p_x%20x%20%2B%20p_y%20y%20%3D%20p_x%20x'%20%2B%20p_y%20y'">, so linear interpolation of one asset implies linear interpolation of the other.
+
+To derive the demand results in the previous section, consider an offer selling asset `X` (we also use `X` to denote the amount of `X` being offered) for asset `Y` at limit price <img style="transform: translateY(0.1em); background: white;" src="https://render.githubusercontent.com/render/math?math=p_%7Blim%7D%3DP">. At any point, the budget must be conserved (subject to a fee `F`):
```

Comment:
There are no fees for orders in this proposal. If you think there should be (I know we discussed this) then you should discuss it on the mailing list.

Reply:
I guess my point is people will think that this means we intend for there to be a fee, or intend the fee to be a future extension. If we don't ever intend that, there's no reason to handle the generality. If we do intend it, then yes let's keep it.

Reply:
I don't intend to propose a fee without a clear reason of doing so (other than possible numerical benefit of having a smaller smooth-factor, which should be pretty minor). 
In that case, I have removed all `F` components from the derivation (except the very first one, to state the general form), and made it clear from the beginning there will be no fee. It also shortens the derivation a bit which is nice.